src/Context.ts

```diff
@@ -87,31 +87,35 @@ class Context implements IContext {
   }
 }
 
-function getLogger(invocationId: string, functionName: string, log: ILogCallback): ILogger{
+function getLogger(invocationId: string, functionName: string, log: LogCallback): Logger{
     return Object.assign(
-      <ILog>(...args: any[]) => log(LogLevel.Information, ...args),
+      <Log>(...args: any[]) => log(LogLevel.Information, ...args),
       {
-        error: <ILog>(...args: any[]) => log(LogLevel.Error, ...args),
-        warn: <ILog>(...args: any[]) => log(LogLevel.Warning, ...args),
-        info: <ILog>(...args: any[]) => log(LogLevel.Information, ...args),
-        verbose: <ILog>(...args: any[]) => log(LogLevel.Trace, ...args)
+        error: <Log>(...args: any[]) => log(LogLevel.Error, ...args),
+        warn: <Log>(...args: any[]) => log(LogLevel.Warning, ...args),
+        info: <Log>(...args: any[]) => log(LogLevel.Information, ...args),
+        verbose: <Log>(...args: any[]) => log(LogLevel.Trace, ...args)
       }
     );
 }
 
-export interface IInvocationResult {
+export interface InvocationResult {
   return: any;
-  bindings: IDict<any>;
+  bindings: Dict<any>;
 }
 
-export interface ILogCallback {
+export interface DoneCallback {

```

Comment:
You can do this as a type - don't need an interface.

export type DoneCallback = (err?: any, result?: any): void;

Also, `any` is dangerous here. Would recommend thinking hard about weather `any` is really acceptable for both parameters.